### PR TITLE
[web] 110 multiple inference

### PR DESCRIPTION
**Pull request by elliotmessi**
one dataset, more models

### CODE REVIEW

**Review by IJtLJZ8Rm4Yr — DISMISSED** [review]

**Review by IJtLJZ8Rm4Yr — DISMISSED** [review]

**Review by fenrir-z — DISMISSED** [review]

File: ymir/web/src/locales/modules/common.ts [hunk 1]
```diff
@@ -66,6 +66,7 @@ const common = {
   'common.action.hide': {cn: '隐藏', en: 'Hide', },
   'common.action.multiple.hide': {cn: '批量隐藏', en: 'Multiple Hide', },
   'common.action.multiple.restore': {cn: '批量恢复', en: 'Multiple Restore', },
+  'common.action.multiple.infer': {cn: '批量推理', en: 'Multiple Infer', },
```
Comment (phoenix-xhuang) [inline ymir/web/src/locales/modules/common.ts]: Batch Inference

File: ymir/web/src/locales/modules/common.ts [hunk 2]
```diff
@@ -66,6 +66,7 @@ const common = {
   'common.action.hide': {cn: '隐藏', en: 'Hide', },
   'common.action.multiple.hide': {cn: '批量隐藏', en: 'Multiple Hide', },
   'common.action.multiple.restore': {cn: '批量恢复', en: 'Multiple Restore', },
```
Comment (phoenix-xhuang) [inline ymir/web/src/locales/modules/common.ts]: Batch Restore/Batch Hide

File: ymir/web/src/locales/modules/task.ts [hunk 3]
```diff
@@ -76,6 +76,8 @@ const task = {
   "task.inference.form.image.placeholder": { cn: "请选择用于推理的镜像", en: "Please select a docker image for inference", },
   "task.inference.form.image.required": { cn: "请选择用于推理的镜像", en: "Please select a docker image for inference", },
   "task.inference.form.model.required": { cn: "请选择用于数据推理的模型", en: "please select the model used for data inference", },
+  "task.inference.model.iters": { cn: "迭代产生的模型", en: "Models from iterations", },
+  "task.inference.failure.some": { cn: "部分推理失败", en: "Some inference failure", },
```
Comment (phoenix-xhuang) [inline ymir/web/src/locales/modules/task.ts]: Part of inference tasks fail

File: ymir/web/src/locales/modules/task.ts [hunk 4]
```diff
@@ -76,6 +76,8 @@ const task = {
   "task.inference.form.image.placeholder": { cn: "请选择用于推理的镜像", en: "Please select a docker image for inference", },
   "task.inference.form.image.required": { cn: "请选择用于推理的镜像", en: "Please select a docker image for inference", },
   "task.inference.form.model.required": { cn: "请选择用于数据推理的模型", en: "please select the model used for data inference", },
+  "task.inference.model.iters": { cn: "迭代产生的模型", en: "Models from iterations", },
```
Comment (phoenix-xhuang) [inline ymir/web/src/locales/modules/task.ts]: Model generated by iteration process

File: ymir/web/src/locales/modules/common.ts (hunk 1 above)
Comment (elliotmessi) [reply ymir/web/src/locales/modules/common.ts]: fixed

File: ymir/web/src/locales/modules/common.ts (hunk 2 above)
Comment (elliotmessi) [reply ymir/web/src/locales/modules/common.ts]: fixed

File: ymir/web/src/locales/modules/task.ts (hunk 3 above)
Comment (elliotmessi) [reply ymir/web/src/locales/modules/task.ts]: fixed

File: ymir/web/src/locales/modules/task.ts (hunk 4 above)
Comment (elliotmessi) [reply ymir/web/src/locales/modules/task.ts]: fixed

**Review by phoenix-xhuang — APPROVED** [review]

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**